src/m2n/tests/PointToPointCommunicationTest.cpp

```diff
@@ -305,6 +305,120 @@ BOOST_AUTO_TEST_CASE(MPIPortsCommunication,
   }
 }
 
+// we check whether connections between participants ranks are build correctly! 
+BOOST_AUTO_TEST_CASE(ConnectionTest, * testing::OnSize(4))
+{
+  
+  assertion(utils::Parallel::getCommunicatorSize() == 4);
+
+  // com::PtrCommunicationFactory cf(new com::SocketCommunicationFactory);
+  com::PtrCommunicationFactory cf(new com::MPIPortsCommunicationFactory);  
```

Comment:
We should test both here. Simply put things in a function and create two tests which both call the function. Similar to the previous tests.

Reply:
done!